src/app/Controllers/ActorsController.cs

```diff
@@ -89,15 +84,15 @@ private string GetMethodText(string q, int pageNumber, int pageSize)
                 // add the query parameters to the method name if exists
                 if (HttpContext.Request.Query.ContainsKey("q"))
                 {
-                    method = string.Format(CultureInfo.InvariantCulture, $"{method}:q:{q}");
+                    method = string.Format(CultureInfo.InvariantCulture, $"{method}:q:{actorQueryParameters.Q}");
```

Comment:
this can be simplified to method = $"{...";

I've been cleaning them up as I touch the code.